pyscriptjs/src/stores.ts

```diff
@@ -1,17 +1,22 @@
 import { writable } from 'svelte/store';
 import type { PyLoader } from './components/pyloader';
 import type { PyScript } from './components/pyscript';
-import type { PyodideInterface } from './pyodide';
+import type { Runtime } from './runtime';
 
 export type Initializer = () => Promise<void>;
 
 export type Environment = {
     id: string;
-    runtime: PyodideInterface;
+    runtime: Runtime;
     state: string;
 };
 
-export const pyodideLoaded = writable<PyodideInterface>();
+/*
+Instead of having a store for Pyodide specifically,
```

Comment:
This should be a generic comment not a reference to pyodide because it makes it harder for a dev without context to understand

Reply:
removed the reference to `pyodide`